### PR TITLE
RDISCROWD-5567 enhance task guidelines images

### PR DESCRIPTION
**Pull request by n00rsy**
- Enable image uploading for task guidelines
- Improve image loading performance compared to previous implementation

Re: https://github.com/bloomberg/pybossa/pull/801

### CODE REVIEW

File: static/js/image_upload.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            contentType: false,
+        }).done(response => {
+            response.imgurls.forEach(imgurl => {
+                let image = $('<img>').attr('src', imgurl).on('load', () => {
+                    image.css('width', Math.min(summernote.width(), this.width))
+                });
+                summernote.summernote("insertNode", image[0]);
+            })
+        }).fail(response => {
+            console.log("upload image failed. response:", response)
```
Comment (primaryobjects) [inline static/js/image_upload.js:30]: Could we set this to a `console.error(...)` instead of log?

**Review by primaryobjects — COMMENTED** [review]
I believe we should undo the changes in this PR to the *.html files that get auto-generated (all .html files except for `task_presenter_editor.html`). Thanks.

File: static/js/image_upload.js (hunk 1 above)
Comment (kbecker42) [inline static/js/image_upload.js:30]: Consider `console.error(...`)`. Thank you.

**Review by kbecker42 — COMMENTED** [review]
I believe we should undo the changes in this PR to the *.html files that get auto-generated (all .html files except for `task_presenter_editor.html`). Thanks.

File: static/js/image_upload.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+const TaskPresenterImageUploader = {
+    MAX_FILE_SIZE_MB: 5,
+    uploadImages: (files, short_name, summernote) => {
+        let fd = new FormData();
+        Array.from(files).forEach(file => {
+            if (file.size / 1024 / 1024 < TaskPresenterImageUploader.MAX_FILE_SIZE_MB) {
+                fd.append("image", file);
+            }
+            else {
+                alert("Images must be less than " + TaskPresenterImageUploader.MAX_FILE_SIZE_MB + " mb.")
```
Comment (XiChenn) [inline static/js/image_upload.js:12]: Minor: Using "MB" instead of "mb". Lowercase 'b' usually is used as "bit".

**Review by kbecker42 — APPROVED** [review]

**Pull request merged**